### ISSUE TITLE
Add Typescript Support

### ISSUE THREAD

**Issue by kolencherry** [twilio-labs/create-twilio-function#52]
We provide [a guide for using TypeScript with Twilio Serverless](https://www.twilio.com/docs/labs/serverless-toolkit/guides/typescript). It would be good to add first-class support for typescript in `create-twilio-function` to make it easier for developers to spin up functions written in TypeScript.

**Comment by philnash** [comment]
I think this is totally do-able.

It won't work if you provide a template (since they are all JS right now), but passing the flag `--typescript` could definitely do the setup and throw in some example `.ts` files.

**Comment by philnash** [comment]
Or should it be `--type-script` ?

**Comment by philnash** [comment]
I found time to do this and it is released as `create-twilio-function` 2.3.0.

Try it now with:

```
npm init twilio-function my-typescript-functions --typescript
```

And enjoy!
